### DISCUSSION TITLE
A question about the add-forward option in the manual

### DISCUSSION THREAD

**Discussion by unknowndevQwQ** [firewalld/firewalld#1402] [Q&A]
It is noted in the manual that ipv6 forwarding requires the use of rich rules, but the nft rules generated by rich rules are the same as the rules generated by the forward option.

**Comment by erig0** [comment]
> noted in the manual that ipv6 forwarding requires the use of rich rules

IPv6 masquerade requires a rich rule.

The `--add-forward` option is about [intra-zone forwarding](https://firewalld.org/2020/04/intra-zone-forwarding). It's applies to both IPv4 and IPV6.
